english/strange-room/room.md

```diff
@@ -4,7 +4,9 @@ It's obvious someone has been here recently but who?
 
 You don't feel scared in the room for now, so you decide to look around for a
 few minutes. You look under the table, and find a card with a random phone
-number written on it.
```

Comment:
I believe I have fixed the issue now. I'm not sure if I need to do anything else to revise the merge request.